### PR TITLE
added git/cli materials

### PR DESCRIPTION
**Pull request by nayana487**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -23,4 +23,14 @@ plan your time accordingly.
...
 ## Part 3: Advanced JavaScript
 
-- Read https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Objects/Object-oriented_JS
\ No newline at end of file
+- Read https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Objects/Object-oriented_JS
+
+## Part 4: Git & Github
+
+- Go through https://www.codecademy.com/learn/learn-git
+- Read https://git-scm.com/doc
```
Comment (jshawl) [inline readme.md:31]: ^^ I feel like this is a lot, maybe just chapters 1 (getting started), 2(basics), and 6(github)?

File: readme.md [hunk 2]
```diff
@@ -23,4 +23,14 @@ plan your time accordingly.
...
+- https://try.github.io/levels/1/challenges/1
+
+## Part 5: Command Line Interface
+
+- Complete http://generalassembly.github.io/prework/cl/#/
```
Comment (jshawl) [inline readme.md:39]: where did you get this link? we should fork it to prevent 404s

File: readme.md (hunk 1 above)
Comment (nayana487) [reply readme.md:31]: 👍 updated with specific chapters

File: readme.md (hunk 2 above)
Comment (nayana487) [reply readme.md:39]: This came from the DSI pre-work materials. Can't find the repo--think it's private. Want me to add something else? It's a pretty good resource though

Comment (jshawl) [reply readme.md:39]: @nayana487 https://ga-wdi-exercises.github.io/prework/cl/#/

**Pull request merged**